packages/bento-frontend/public/injectEnv.js

```diff
@@ -15,8 +15,8 @@ window.injectedEnv = {
   // IDP Auth url
   REACT_APP_NIH_AUTH_URL: 'https://stsstg.nih.gov/auth/oauth/v2/authorize',
   // Client IDs for IDP
-  REACT_APP_GOOGLE_CLIENT_ID: 'Sample id',
-  REACT_APP_NIH_CLIENT_ID: 'Sample id',
+  REACT_APP_GOOGLE_CLIENT_ID: '196014713877-0d926jpdd691roubuc0kpu6r6ha9b9t5.apps.googleusercontent.com',
```

Comment:
Can you remove client Ids values? and replace it with 'Sample id'? We do usually do not commit client id values in github.